app/webpacker/styles/course_education_phase_form.scss

```diff
@@ -0,0 +1,5 @@
+.course-education-phase-form {
```

Comment:
You can use the `govuk-!-font-weight-regular` class to override this directly in the markup

Reply:
Good news. We can now get rid of the CSS stuff because the latest version of `govuk_design_system_formbuilder` now gives us the option to turn off bold labels.

Reply:
This ideally shouldn't be needed.